inst/odin/carehomes.R

```diff
@@ -1511,18 +1510,15 @@ update(react_pos) <- sum(new_T_PCR_pos[2:18, , , ])
 
 ## grp_prob_strain is probability of an infection in group i being of strain j
 ## prob_strain is probability of an infection being of strain j
-grp_prob_strain[, ] <- lambda[i, j] / sum(lambda[i, ])
-dim(grp_prob_strain) <- c(n_groups, n_real_strains)
+grp_prob_strain[, , ] <- lambda_susc[i, j, k] / sum(lambda_susc[i, , k])
```

Comment:
it would be good to discuss this with @edknock. I am not sure we want to weigh this by the relative susceptibility.

Reply:
actually I'm pretty sure this should be using lambda not lambda_susc as it ignores how many susceptible are in each vaccination class